CHANGELOG.md

```diff
@@ -218,6 +231,9 @@
 
 ### Features
 - show Sigstore signatures on kwctl inspect
+- show Sigstore signatures on kwctl inspect
+- show Sigstore signatures on kwctl inspect
```

Comment:
this block is repeated. I wonder if it is because of commits actually. Anyway! I don't care anymore xD.

Reply:
It's the changelog genator, this should really be sorted out :/